packages/core/src/context/experiment/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './store'
 export * from './experiment-context'
 export * from './experiment-selectors'
+export * from './experiment-reducers'
```

Comment:
I think experiment reducer is not exposed intentionally to force clients to use "rootReducer" only when overriding functionality.